integrations/lib/multiwoven/integrations/destination/zendesk/client.rb

```diff
@@ -0,0 +1,124 @@
+module Multiwoven
+    module Integrations
+        module Destination
+            module Zendesk
+                include Multiwoven::Integrations::Core
+
+                API_VERSION = "59.0"
+
+                class Client < DestinationConnector
+                    prepend Multiwoven::Integrations::Core::RateLimiter
+                    def check_connection(connection_config)
+                        connection_config = connection_config.with_indifferent_access
+                        initialize_client(connection_config)
+                        authenticate_client
+                        success_status
+                      rescue StandardError => e
+                        handle_exception("ZENDESK:CHECK_CONNECTION:EXCEPTION", "error", e)
+                        failure_status(e)
+                    end
+
+                    def discover(_connection_config = nil)
+                        catalog = build_catalog(load_catalog)
+                        catalog.to_multiwoven_message
+                        rescue StandardError => e
+                            handle_exception("ZENDESK:DISCOVER:EXCEPTION", "error", e)
+                            failure_status(e)
+                    end
+
+                    def write(sync_config, tickets, action = "create")
```

Comment:
nitpick: rename tickets to records. All the write method signature we are using records

Reply:
This is done.